src/index.js

```diff
@@ -15,6 +16,9 @@ import ERRORS from './errors';
 
 import testRunTracker from 'testcafe/lib/api/test-run-tracker';
 
+const DEBUG_LOGGER = debug('testcafe:electron');
```

Comment:
It's a great idea to use `testcafe` as a top namespace, but I would like to use `browser-provider-electron` on the second level to be consistent with the package name.

Reply:
willfix.